packages/data-stores/src/reader/queries/use-post-subscriptions-query.ts

```diff
@@ -60,7 +60,7 @@ const usePostSubscriptionsQuery = ( {
 		// Transform the dates into Date objects
 		const transformedData = flattenedData?.map( ( comment_subscription ) => ( {
 			...comment_subscription,
-			subscription_date: new Date( comment_subscription.subscription_date ),
+			subscription_date: new Date( comment_subscription.subscription_date + ' UTC' ),
```

Comment:
Would you keep this change in the frontend - or would you address it on the endpoint side? I see benefits in both approaches, but would be interested in hearing your point of view.

Reply:
Frontend. This gives room for endpoint to be cached, if the user flies from NL to BG on the same day and the hour changes.

Reply:
But in both cases (implementation in front-end vs back-end), the `new Date()` is still getting the same input string. The difference is where the `UTC` is appended to the end of that string.

Reply:
Ah you're talking about appending the `UTC` string on the frontend or backend (not about the timezone offsets). 
I would append `UTC` string on the back end since this is something that `new Date()` is expecting.

Reply:
In that case, I will leave the UTC string on the frontend side - just so we can move forward with the deployment of the Comments tab and then will remove it in the later PR, while also adding the UTC string on the endpoint.